WeaveApp/webpack.prod.js

```diff
@@ -0,0 +1,23 @@
+const merge = require('webpack-merge');
+const UglifyJSPlugin = require('uglifyjs-webpack-plugin');
+const Visualizer = require('webpack-visualizer-plugin');
+const common = require('./webpack.common.js');
+const webpack = require('webpack');
+
+module.exports = merge(common, {
+    devtool: 'source-map',
+    plugins: [
+        new UglifyJSPlugin({
+            sourceMap: true,
+        }),
+        new webpack.DefinePlugin({
```

Comment:
This defines the node environment as production, which has a number of performance gains for libraries we use, specifically react.